src/manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "description": "WebExtension for Motrix download manager",
+  "manifest_version": 2,
+  "name": "Moxtrix WebExtension",
+  "version": "2.0.0",
+  "homepage_url": "https://github.com/gautamkrishnar/motrix-chrome-extension/",
+  "icons": {
+    "16": "assets/images/16.png",
+    "32": "assets/images/32.png",
+    "48": "assets/images/48.png",
+    "128": "assets/images/128.png"
+  },
+  "permissions": [
+    "downloads",
+    "storage",
+    "notifications"
+  ],
+  "browser_action": {
+    "browser_style": true,
+    "default_icon": {
+      "16": "assets/images/16.png",
+      "32": "assets/images/32.png",
+      "48": "assets/images/48.png",
+      "128": "assets/images/128.png"
+    },
+    "default_title": "Motrix WebExtension",
+    "default_popup": "popup.html"
+  },
+  "background": {
+    "scripts": [
+      "browser-polyfill.min.js",
+      "background.js"
+    ],
+    "persistent": true
+  },
+  "browser_specific_settings": {
```

Comment:
@gautamkrishnar Chrome doesn't support "browser_specific_settings", therefore before publishing to a specific browser add and remove this accordingly. I will create a publishing script later.

Reply:
@149segolte it is taken from https://stackoverflow.com/questions/8667070/javascript-regular-expression-to-validate-url

Feel free to use any. I think checking http/https and :// would be enough. You can use any other regex mentioned in the Stackoverflow.